libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -318,7 +318,7 @@ RC_Channel::percent_input() const
 }
 
 /*
-  Return true if the channel is at trim and within the DZ
+  Return true if the channel is within the DZ
```

Comment:
The current comment isn't great but the new comment is worse in my opinion, it doesn't specify which dead zone it is - which in this method is specifically a dead zone around the trim value.